src/Loader/MoLoader.php

```diff
@@ -125,9 +125,7 @@ private function seekTo(int $position): void
 
     private function readInt(string $byteOrder): int
     {
-        if (($read = $this->read(4)) === false) {
-            return 0;
-        }
+        $read = $this->read(4);
```

Comment:
I'm not sure if this is equivalent.
If the `$read` variable is `false`, it returns `0`. The new code doesn't check this condition.

Reply:
That's the neat part, the signature of that function is `private function read(int $bytes): string`, and PHPStan reports that if as `MoLoader::read() does never return false`